### PR TITLE
Add optional images to section links

### PR DESCRIPTION
**Pull request by chrisrhymes**
- Update storybook
- Add optional images to SectionLinks using LazyImage
- Extract pageLinksArray to a storydata file
- Add tests for SectionLinks
- Fix docs not displaying correctly for SectionLinks
- Add dependabot.yml file

### CODE REVIEW

File: src/library/structure/SectionLinks/SectionLinks.styles.js [hunk 1]
```diff
@@ -29,69 +30,91 @@ export const LinksList = styled.div`
...
 
 export const Summary = styled.p`
   margin: 0;
-  color: ${props => props.theme.theme_vars.colours.black} !important;
+  color: ${(props) => props.theme.theme_vars.colours.black} !important;
   font-weight: 400;
-`
\ No newline at end of file
+`;
+
```
Comment (chrisrhymes) [inline src/library/structure/SectionLinks/SectionLinks.styles.js:103]: Everything above is prettier formatting. This is the addition below.

File: src/library/structure/SectionLinks/SectionLinks.tsx [hunk 2]
```diff
@@ -1,37 +1,42 @@
...
-                <Styles.Pagelink 
-                    key={i} 
-                    href={link.url} 
-                    title={link.title}
+  <Styles.Container id={sectionSlug} data-testid="SectionLinks">
+    {displayTitle ? <Styles.SectionTitle text={sectionTitle} /> : <br />}
+    <Styles.LinksList>
+      {pageLinksArray.map((link, i) => (
+        <Styles.Pagelink key={i} href={link.url} title={link.title}>
+          {hasImages && (
```
Comment (chrisrhymes) [inline src/library/structure/SectionLinks/SectionLinks.tsx:18]: Mostly prettier, but everything inside the hasImages check is new

File: src/library/structure/SectionLinks/SectionLinks.types.ts [hunk 3]
```diff
@@ -1,37 +1,63 @@
-
 export interface SectionLinksProps {
   /**
    * The title of the section
    */
   sectionTitle: string;
+
   /**
    * If there is only one section then do not dieplay the title
```
Comment (DamienBurkeWN) [inline src/library/structure/SectionLinks/SectionLinks.types.ts:8]: dieplay? 😆

**Review by DamienBurkeWN — APPROVED** [review]
All good, possible exception being display on mobile when some links don't have images - the blank space is more of an issue then than on desktop I think? In practice I can't see them setting things up like that though?

**Review by DamienBurkeWN — APPROVED** [review]
Sweet 👍

**Pull request merged**